dbms/src/Flash/Pipeline/Schedule/ThreadPool/TaskThreadPoolImpl.cpp

```diff
@@ -27,6 +28,10 @@ TaskQueuePtr CPUImpl::newTaskQueue(TaskQueueType type)
     {
     // the default queue is mlfq queue.
     case TaskQueueType::DEFAULT:
+#ifdef DBMS_PUBLIC_GTEST
+    case TaskQueueType::RCQ_MLFQ:
+        return std::make_unique<ResourceControlQueue<CPUMultiLevelFeedbackQueue>>();
+#endif
```

Comment:
If the ResourceGroupQueue is temporarily unavailable, it can be set to non-default?

Reply:
done